pkgs/applications/audio/littlegptracker/default.nix

```diff
@@ -1,75 +1,63 @@
 { lib, stdenv
 , fetchFromGitHub
-, unstableGitUpdater
-, SDL
+, SDL2
+, python3
 , jack2
 , Foundation
+, alsa-lib
+, pkg-config
 }:
-
-stdenv.mkDerivation {
+let
+  python = python3.withPackages (ps: with ps; [ pillow ]);
+  platform = if stdenv.hostPlatform.isLinux then "X64" else "OSX";
+in stdenv.mkDerivation rec {
   pname = "littlegptracker";
-  version = "0-unstable-2020-11-26";
+  version = "1.4.2";
 
   src = fetchFromGitHub {
-    owner = "Mdashdotdashn";
+    owner = "djdiskmachine";
     repo = "littlegptracker";
-    rev = "4aca8cd765e1ad586da62decd019e66cb64b45b8";
-    sha256 = "0f2ip8z5wxk8fvlw47mczsbcrzh4nh1hgw1fwf5gjrqnzm8v111x";
+    rev = "refs/tags/${version}";
+    hash = "sha256-1uXC5nJ63YguQuNIkuK0yx9lmrMBqw0WdlmCV8o11cE=";
   };
 
   buildInputs = [
-    SDL
+    pkg-config
+    SDL2
+    python
```

Comment:
pkg-config and python are build-time dependencies and should go in `nativeBuildInputs`